### ISSUE TITLE
GROUPS: UX review needed for invites related to adding topics to a group  

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1665]
**What is the problem?**
As noted in [this user insights issue](https://github.com/citizenos/citizenos-fe/issues/1548
), the UX of adding a topic to a group and what this means in terms of invites being sent needs some thinking through from the users' perspective. See the issue for a summary of the user insights.  

Also, as noted by @ilmartyrk in [this comment](https://github.com/citizenos/citizenos-fe/issues/1548#issuecomment-2355227645) in the same issue, we currently send out emails saying 'User X has invited you to join topic X' when actually they've not been invited as such, it's rather that the topic has been added to a group they are in.

**Why is this a problem?**
Was shown to cause confusion for admins in a user test - uncertainty whether the members of their group have been sent invitations and what that invitation said. Could also cause confusion for participants thinking they have been directly invited to a topic, when actually the topic has only been added to a group they're in, which isn't always the same thing. 

**Possible solution.**
Needs a UX review. A few things in particular needing consideration: 
- Should admins be informed during the create/edit flow that invites will be sent to the group members once they add it to a group? 
- Should there be an option to add a bespoke text to notifications to members of the group when you add a topic to the group, like is possible when you invite people directly to a topic? 
- Should admins still be prompted to invite people after publishing, if they have already added it to a group? Or if so, should it be clearer at that point that the group has been invited? 
- Should we have a different kind of email notification when topics get added to a group you are in? 

Note that in [this issue](https://github.com/citizenos/citizenos-fe/issues/1548), we are adding notifications that group invites have been sent, as a 'quick fix'.

Will put this to 'soon' and assign @kevincrepin, @Lidziya-p and @TiinaDUX for input :)

**Comment by kevincrepin** [comment]
My thoughts:

**Should admins be informed during the create/edit flow that invites will be sent to the group members once they add it to a group?**
I think we can say this, but I would add it to the text that's already there. Not make it into a separate notification or smt. 
![Screenshot 2024-11-15 at 13 58 34](https://github.com/user-attachments/assets/bcceda74-02f6-476a-845d-d32703b6cb46)

**Should there be an option to add a bespoke text to notifications to members of the group when you add a topic to the group, like is possible when you invite people directly to a topic?**
No problem with that. Considering the admin can add multiple groups it's something we need to keep in mind. Probably per group the admin can write a different message.
![Screenshot 2024-11-15 at 14 00 19](https://github.com/user-attachments/assets/b6e59e2a-8365-4d09-a2f1-06cbd6adfd6f)

**Should admins still be prompted to invite people after publishing, if they have already added it to a group? Or if so, should it be clearer at that point that the group has been invited?**
I would still keep this prompt, same behaviour across the lone, otherwise it's an exception based on whether you added a group or not. Still possible you want to invite people outside of the group.

**Should we have a different kind of email notification when topics get added to a group you are in?**
I think it should be a separate e-mail yes, but probably we can re-use the template we have for joining a topic. I would just update the top sentence to "A new topic has been added to group [group name]". Wdyt @BeccaMelhuish ? We can also bring out the group more if needed. 
![Screenshot 2024-11-15 at 14 05 31](https://github.com/user-attachments/assets/2d32d4ac-8792-4577-86a2-296f5902d8f2)

**Comment by BeccaMelhuish** [comment]
@kevincrepin I think this all seems great, though a few thoughts: 

- Re the top one, we do now have a lot more text there already plus a tooltip about group rights which came about following some user issues. Do you think adding this text there too would be too much? Maybe it could go under 'Groups you're adding this topic to' instead? 

![image](https://github.com/user-attachments/assets/16540597-408a-44f8-a81d-32deaf19ab60)

- Re the bottom one, looks great! Though how about just this for the text below, rather than saying they've been invited? 

_Participant X has added a new topic to group X.' 

And should the group be blue and linked? 

![image](https://github.com/user-attachments/assets/26904e3e-ba12-4cc1-b585-3c873b0a4aac)

**Comment by kevincrepin** [comment]
A lot of text indeed. OK, let's add it like you suggested. I think in general there's too much text here. Ideally it's shorter with maybe a text link "more about adding topics to groups" which would open an entire topic on this in the help centre.

Ok with the text change for the email. Not sure if the group needs to be clickable, because it's more about the topic than the group, no? It can be though, not a massive thing, I'd say :)

**Comment by BeccaMelhuish** [comment]
@kevincrepin yes true that text has become a bit overwhelming! We don't actually have a help centre at the moment.. But I will make a separate issue to think this through in - perhaps we start bringing back the HC in minimal format for such cases. There are probably a few other places where similar would be useful :) 

OK good, yes true - maybe linked group isn't necessary!
